mwdb/web/src/components/Settings/Views/UsersList.js

```diff
@@ -52,9 +53,11 @@ function UserItem(props) {
             </td>
             <td>{feed_badge}</td>
             <td>
-                {groups.map((group) => (
-                    <GroupBadge group={group} clickable basePath="/admin" />
-                ))}
+                <LimitTo count={5}>
+                    {groups.map((group) => (
+                        <GroupBadge group={group} clickable basePath="/admin" />
+                    ))}
```

Comment:
We can add filter here to prevent repetition of user login in groups
```suggestion
                    {groups
                        .filter((group) => group.name !== props.login)
                        .map((group) => (
                            <GroupBadge
                                group={group}
                                clickable
                                basePath="/admin"
                            />
                        ))}
```